.ci/jenkins/dsl/jobs.groovy

```diff
@@ -0,0 +1,38 @@
+import org.kie.jenkins.jobdsl.templates.KogitoJobTemplate
+
+def getDefaultJobParams(String repoName = 'optaweb-vehicle-routing') {
+    return KogitoJobTemplate.getDefaultJobParams(this, repoName)
+}
+
+Map getMultijobPRConfig() {
+    return [
+        parallel: true,
+        buildchain: true,
+        jobs : [
+            [
+                id: 'optaweb-vehicle-routing',
+                primary: true,
+            ]
+        ],
+    ]
+}
+
+setupMultijobPrDefaultChecks()
```

Comment:
The same concern as here: https://github.com/kiegroup/optaweb-employee-rostering/pull/677#discussion_r691988963